src/amo/routes.js

```diff
@@ -14,6 +15,7 @@ export default (
     <IndexRoute component={Home} />
     <Route path="addon/:slug/" component={DetailPage} />
     <Route path="addon/:slug/review/:reviewId/" component={AddonReview} />
+    <Route path="categories/:addonType/" component={CategoriesPage} />
```

Comment:
If we're aiming for 1:1 URL parity with the old AMO (which I wasn't ever clear on, but I guess we are) then yes.

I guess since the new site will switch over via cookies (and cool URLs don't change) that's best.